Providers/Modules/Plugins/ChangeTracking/plugin/change_tracking_runner.rb

```diff
@@ -33,28 +32,37 @@ def self.transform_and_wrap()
 			file = File.open(CHANGE_TRACKING_FILE, "rb")
 			xml_string = file.read; nil # To top the output to show up on STDOUT.
 
+			previousSnapshot = ChangeTracking.getHash(CHANGE_TRACKING_STATE_FILE)
+                        previous_inventory_checksum = {}
```

Comment:
nit: there seem to be multiple places here where indentation should be cleaned up